roachtest: activerecord failed

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18263974?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18263974?buildTab=artifacts#/activerecord) on master @ [0806ee44e36b62dce75175702b5bfb3db03e0577](https://github.com/cockroachdb/cockroach/commits/0806ee44e36b62dce75175702b5bfb3db03e0577):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.1.0-alpha.2-dev-0806ee44e36b62dce75175702b5bfb3db03e0577
Tests run against activerecord 7.2.1
8482 Total Tests Run
8480 tests passed
2 tests failed
60 tests skipped
10 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: FixtureWithSetModelClassPrevailsOverNamingConventionTest#test_model_class_in_fixture_file_is_respected (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18263974/activerecord/1734763753333/1734772646368)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #136675 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-3 T-sql-foundations branch-release-24.2.6-rc]
- #136561 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-3 T-sql-foundations branch-release-24.3.1-rc]
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-45788

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18268945?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18268945?buildTab=artifacts#/activerecord) on master @ [01f9e61532862cbbdbc64180013ca1cb57f4a017](https://github.com/cockroachdb/cockroach/commits/01f9e61532862cbbdbc64180013ca1cb57f4a017):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.1.0-alpha.2-dev-01f9e61532862cbbdbc64180013ca1cb57f4a017
Tests run against activerecord 7.2.1
8482 Total Tests Run
8480 tests passed
2 tests failed
60 tests skipped
10 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: FixtureWithSetModelClassPrevailsOverNamingConventionTest#test_model_class_in_fixture_file_is_respected (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18268945/activerecord/1734850072270/1734858770294)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #136675 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-3 T-sql-foundations branch-release-24.2.6-rc]
- #136561 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-3 T-sql-foundations branch-release-24.3.1-rc]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18285265?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18285265?buildTab=artifacts#/activerecord) on master @ [d58f071217505b226a097d39f87072f4e1bd8e06](https://github.com/cockroachdb/cockroach/commits/d58f071217505b226a097d39f87072f4e1bd8e06):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.1.0-alpha.2-dev-d58f071217505b226a097d39f87072f4e1bd8e06
Tests run against activerecord 7.2.1
8482 Total Tests Run
8480 tests passed
2 tests failed
60 tests skipped
10 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: FixtureWithSetModelClassPrevailsOverNamingConventionTest#test_model_class_in_fixture_file_is_respected (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=leader</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18285265/activerecord/1735022734903/1735030905570)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #136675 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-3 T-sql-foundations branch-release-24.2.6-rc]
- #136561 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-3 T-sql-foundations branch-release-24.3.1-rc]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18292427?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18292427?buildTab=artifacts#/activerecord) on master @ [270ba1e8fe656fab0a182643fb77a6a5be64c1b0](https://github.com/cockroachdb/cockroach/commits/270ba1e8fe656fab0a182643fb77a6a5be64c1b0):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.1.0-alpha.2-dev-270ba1e8fe656fab0a182643fb77a6a5be64c1b0
Tests run against activerecord 7.2.1
8482 Total Tests Run
8480 tests passed
2 tests failed
60 tests skipped
10 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: FixtureWithSetModelClassPrevailsOverNamingConventionTest#test_model_class_in_fixture_file_is_respected (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18292427/activerecord/1735109155658/1735116654420)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #136675 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-3 T-sql-foundations branch-release-24.2.6-rc]
- #136561 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-3 T-sql-foundations branch-release-24.3.1-rc]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18299182?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18299182?buildTab=artifacts#/activerecord) on master @ [270ba1e8fe656fab0a182643fb77a6a5be64c1b0](https://github.com/cockroachdb/cockroach/commits/270ba1e8fe656fab0a182643fb77a6a5be64c1b0):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.1.0-alpha.2-dev-270ba1e8fe656fab0a182643fb77a6a5be64c1b0
Tests run against activerecord 7.2.1
8482 Total Tests Run
8480 tests passed
2 tests failed
60 tests skipped
10 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: FixtureWithSetModelClassPrevailsOverNamingConventionTest#test_model_class_in_fixture_file_is_respected (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18299182/activerecord/1735195571779/1735204827641)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #136675 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-3 T-sql-foundations branch-release-24.2.6-rc]
- #136561 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-3 T-sql-foundations branch-release-24.3.1-rc]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18308222?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18308222?buildTab=artifacts#/activerecord) on master @ [a0faa4e779ac9e54d17e8400141b0aa472887974](https://github.com/cockroachdb/cockroach/commits/a0faa4e779ac9e54d17e8400141b0aa472887974):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.1.0-alpha.2-dev-a0faa4e779ac9e54d17e8400141b0aa472887974
Tests run against activerecord 7.2.1
8482 Total Tests Run
8480 tests passed
2 tests failed
60 tests skipped
10 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: FixtureWithSetModelClassPrevailsOverNamingConventionTest#test_model_class_in_fixture_file_is_respected (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18308222/activerecord/1735282198571/1735290452159)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #136675 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-3 T-sql-foundations branch-release-24.2.6-rc]
- #136561 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-3 T-sql-foundations branch-release-24.3.1-rc]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/cmd/roachtest/tests/activerecord_blocklist.go
+++ b/pkg/cmd/roachtest/tests/activerecord_blocklist.go
@@ -26,6 +26,7 @@ var activeRecordIgnoreList = blocklist{
 	`ActiveRecord::ConnectionAdapters::PostgreSQLAdapterTest#test_translate_no_connection_exception_to_not_established`:                                        "pg_terminate_backend not implemented",
 	`BasicsTest#test_default_values_are_deeply_dupped`:                                                                                                         "flaky",
 	`CockroachDB::FixturesTest#test_create_fixtures`:                                                                                                           "flaky",
+	`FixtureWithSetModelClassPrevailsOverNamingConventionTest#test_model_class_in_fixture_file_is_respected`:                                                   "flaky",
 	`TestAutosaveAssociationOnAHasAndBelongsToManyAssociation#test_should_not_save_and_return_false_if_a_callback_cancelled_saving_in_either_create_or_update`: "flaky",
 	`TestAutosaveAssociationOnAHasAndBelongsToManyAssociation#test_should_not_update_children_when_parent_creation_with_no_reason`:                             "flaky",
 	`TestAutosaveAssociationOnAHasAndBelongsToManyAssociation#test_should_update_children_when_autosave_is_true_and_parent_is_new_but_child_is_not`:            "flaky",

